src/main/java/frc/utils/calibration/autoPIDtuner/ControllerType.java

```diff
@@ -0,0 +1,7 @@
+package frc.utils.calibration.autoPIDtuner;
+
+public enum ControllerType {
```

Comment:
consider adding PD_CONROLLER

Reply:
that is the only type of controller that cannot be tuned automatically